client/src/components/LinkDrawer.tsx

```diff
@@ -1,12 +1,17 @@
 import { Link } from 'react-router-dom';
 import { AiOutlineClose } from 'react-icons/ai';
+import UserContext from './UserContext';
+import { UserContextValues } from '../lib/types';
+import { useContext } from 'react';
 
 type props = {
   isOpen: boolean;
   onClick: VoidFunction;
 };
 
 export default function LinkDrawer({ isOpen, onClick }: props) {
+  const context = useContext<UserContextValues>(UserContext);
```

Comment:
I think TS will infer `UserContextValues` from the fact that you are using `UserContext`, so it's not necessary to specify the generic type here.